src/PaginationLink.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  'aria-label': PropTypes.string,
+  children: PropTypes.node,
+  className: PropTypes.string,
+  next: PropTypes.bool,
+  previous: PropTypes.bool,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  tag: 'a',
+};
+
+const PaginationLink = (props) => {
+  const {
+    children,
+    className,
+    next,
+    previous,
+    tag: Tag,
+    ...attributes,
+  } = props;
+
+  const classes = classNames(
+    className,
+    'page-link'
+  );
+
+  let defaultAriaLabel;
+  if (previous) {
+    defaultAriaLabel = 'Previous';
+  } else if (next) {
+    defaultAriaLabel = 'Next';
+  }
+  const ariaLabel = props['aria-label'] || defaultAriaLabel;
+
+  let defaultCaret;
+  if (previous) {
+    defaultCaret = <span>&laquo;</span>;
+  } else if (next) {
+    defaultCaret = <span>&raquo;</span>;
+  }
+
+  return (
+    <Tag
+      {...attributes}
+      className={classes}
+      aria-label={ariaLabel}
+    >
+      {(previous || next) ?
+        (<span>
```

Comment:
extra span here too. you could push the child content to an array to get around the need for the wrapper.